src/main/kotlin/team/jeonghokim/zakbu/domain/auth/service/CustomOauth2UserService.kt

```
@@ -0,0 +1,58 @@
+    private val userRepository: UserRepository
+): DefaultOAuth2UserService() {
+    @Transactional
+    @Throws(OAuth2AuthenticationException::class)
+    override fun loadUser(userRequest: OAuth2UserRequest): OAuth2User {
+        val oauth2User: OAuth2User = super.loadUser(userRequest)
+
+        try {
+            return process(userRequest, oauth2User)
+        } catch (e: AuthenticationException) {
+            throw e
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)